src/widgets/buttons.ts

```diff
@@ -24,18 +26,25 @@ const dragAndDrop = require('./dragAndDrop')
 const cancelIconURI = UI.icons.iconBase + 'noun_1180156.svg' // black X
 const checkIconURI = UI.icons.iconBase + 'noun_1180158.svg' // green checkmark; Continue
 
-function getStatusArea (context) {
-  var box = context.statusArea || context.div || null
+export type StatusAreaContext = {
+  statusArea?: HTMLElement
+  div?: HTMLElement
+  dom?: HTMLDocument
+}
+function getStatusArea (context?: StatusAreaContext) {
+  var box = (context && context.statusArea) || (context && context.div) || null
   if (box) return box
-  let dom = context.dom
+  let dom = context && context.dom
   if (!dom && typeof document !== 'undefined') {
     dom = document
   }
   if (dom) {
     var body = dom.getElementsByTagName('body')[0]
-    box = dom.createEvent('div')
-    body.insertBefore(box, body.firstElementChild)
-    context.statusArea = box
+    ;(box as unknown as HTMLElement) = dom.createElement('div')
```

Comment:
Having to use 'as' will be something we look to change when we refactor to Typescript completely?

Reply:
Ah actually there the cast was unnecessary.

In other places I cast from `HTMLElement | null` to `HTMLElement` to indicate that the code ignores the possibility of `document.createElement('div')` returning null.